nexus-writer/src/nexus/hdf5/event_run_file.rs

```diff
@@ -90,12 +90,12 @@ impl EventRun {
             s![self.num_messages..(self.num_messages + 1)],
         )?;
 
-        let timestamp = Into::<DateTime<Utc>>::into(
+        let timestamp = TryInto::<DateTime<Utc>>::try_into(
```

Comment:
Why is `try_into` being called like this? Can this use the more standard `.try_into()` instead?